### PR TITLE
[IMP] reduce noise in python generated code

### PR DESCRIPTION
**Pull request by nda-odoo**

### CODE REVIEW

File: runbot_testing_recording/controllers/main.py [hunk 1]
```diff
@@ -384,12 +401,30 @@ def replace_idtoxml(model_name, values, args_to_replace):
...
+                        xml_id = get_xml_id(res_id, field.comodel_name)
+                        if xml_id:
+                            key = len(args_to_replace)
+                            args_to_replace[key] = 'self.env.ref(\'%s\').id' % (xml_id, )
+                            comodel_new_values.append('FIELD_%s_TO_REPLACE' % key)
+                        else:
+                            comodel_new_values.append(res_id)
+                    magic_tuple[2] = comodel_new_values
+
+        if field.type == 'many2one':
```
Comment (TheVonyo) [inline runbot_testing_recording/controllers/main.py:422]: consider using elif ?

Comment (nda-odoo) [reply runbot_testing_recording/controllers/main.py:422]: i will consider

File: runbot_testing_recording/models/record.py [hunk 2]
```diff
@@ -49,7 +51,12 @@ def open_registration(self):
   
     def start_recording(self):
         self.ensure_one()
-        content = '\'\'\'\n%s\n\'\'\'' % (self.description) if self.record_type == 'test' else '<!--\n%s\n-->' % (self.description)
+        if self.record_type == 'test' :
+            user_decorator = '@users("%s")' % self.env.user.login if self.single_user else ''
+            method_name = 'def test_%s(self):' % self.technical_name
+            content = '%s\n%s\n    \'\'\'\n%s\n    \'\'\'' % (user_decorator, method_name, self.description if self.description else '')
```
Comment (TheVonyo) [inline runbot_testing_recording/models/record.py:57]: use \t for tab ?

File: runbot_testing_recording/controllers/main.py [hunk 3]
```diff
@@ -238,7 +237,10 @@ def append_call(variable_name, element, todo, replace_in_context):
     return format_call_stack(stack_pre_call, stack_post_call, method_call)
 
 def format_call_stack(stack_pre_call, stack_post_call, method_call):
-    call = method_call
+    indent = '    '
```
Comment (TheVonyo) [inline runbot_testing_recording/controllers/main.py:240]: use \t for tab ?

File: runbot_testing_recording/wizards/test_start.py [hunk 4]
```diff
@@ -11,7 +11,7 @@ class RunbotRecordingTest(models.TransientModel):
     def save(self):
         recording_id = int(self.env['ir.config_parameter'].get_param('runbot.record.current', '0'))
         recording = self.env['runbot.record'].search([('id','=', recording_id)])
-        description = '\'\'\'TODO: DO A TEST HERE: \n %s \n\'\'\'' % (self.description)
+        description = '    \'\'\'TODO: ADD AN ASSERT HERE: \n %s \n    \'\'\'' % (self.description)
```
Comment (TheVonyo) [inline runbot_testing_recording/wizards/test_start.py:14]: use \t for tab ?

File: runbot_testing_recording/models/record.py (hunk 2 above)
Comment (sts-odoo) [reply runbot_testing_recording/models/record.py:57]: only if \t is equivalent to 4 spaces?

**Pull request merged**